src/Microsoft.OData.Core/Batch/ODataBatchWriter.cs

```diff
@@ -410,6 +410,17 @@ public Task FlushAsync()
         /// </summary>
         public abstract void StreamDisposed();
 
+        /// <summary>
+        /// This method is called asynchronously to notify that the content stream of a batch operation has been disposed.
+        /// </summary>
+        /// <returns>A task that represents the asynchronous operation.</returns>
+        public virtual Task StreamDisposedAsync()
+        {
+            // NOTE: ODataBatchWriter class is abstract and public. This method body is intended
+            // to prevent a breaking change in subclasses that provide the implementation.

```

Comment:
I think this message is a bit confusing. I think it would be clearer to say something like:
`This method body is intended a breaking change in subclasses that do not override this method`.
When you say "subclasses that provide the implementation", it's not clear whether you're referring to the implementation of the abstract class, or implementation of the `StreamDisposeAsync` method.